ui/src/helpers/api.js

```diff
@@ -46,6 +46,99 @@ export default class Api {
         return response.data;
     }
 
+    //#region User api
+
+    getUsers = async(filter = "") =>{
+        const response = await this.client.get(`/user/listUsers?usernameFilter=${filter}`);
+        return response.data;
+    }
+
+    getUserDetails = async(user) => {
+        const response = await this.client.get(`/user/${user.userId}`);
+        return response.data;
+    }
+
+    updateUser = async(user) => {
+        const response = await this.client.put(`/user/${user.userId}`,user);
+        return response.data;
+    }
+
+    deleteUser = async(user) => {
+        const response = await this.client.delete(`/user/${user.userId}`);
+        return response.data;
+    }
+
+    recoverUser = async(data) => {
+        const form = new FormData();
```

Comment:
why do you need the FormData object?
can't it be just:
const {password, inviteToken} = data;
const response = await this.client.post(`/user/recover`,{password, inviteToken});

Reply:
I asked rami,
server requements